### ISSUE TITLE
Repository links

### ISSUE THREAD

**Issue by AaronLasseigne** [rubygems/rubygems.org#906]
Is "Wiki" a particularly useful link to have? A wiki doing what? A wiki is how something is implemented and not what it is. My guess is that it's for wiki style documentation. If that's the case it seems to me like it would be more useful to have "Documentation" and "Guides" as URLs.

Additionally, there is a place for a mailing list but nothing for chat. Many projects have a way for you to chat with the developers or other users via HipChat, IRC, etc. It might be worth adding.

**Comment by arthurnn** [comment]
We have the wiki and http://guides.rubygems.org/ .

I am usually on IRC, and I would say the #rubygems on freenode is the best way to get in touch with the rubygems devs.

**Comment by AaronLasseigne** [comment]
I think there was a misunderstanding. I wasn't clear enough in my issue. I'm referencing the links a gem owner can add.

**Comment by AaronLasseigne** [comment]
@arthurnn Can I get a response to the clarification I made or get the ticket re-opened?

**Comment by AaronLasseigne** [comment]
thanks

**Comment by qrush** [comment]
I added `wiki` a while back as one of the first commits in 2009: 50bc59040ae3876269e3311ee8ba064cc1471ff4

As far as I can recall, the `Linkset` table's purpose was trying to link outside of Gemcutter to external services that RubyForge or GitHub may have provided before. Some projects had a wiki. I thought it might be useful.

Fast forward 6+ years: There's plenty of links to consider now. I'd love to see a proposal to use the `metadata` field to allow arbitrary linking, or categories of links..and to impose some reasonable restrictions.

http://guides.rubygems.org/specification-reference/#metadata

Closing this issue for now - feel free to bring a new one up with a new way to handle links!
